tools/install-powershell.sh

```diff
@@ -120,42 +125,45 @@ install(){
     echo "  REV: $REV"
     echo "  KERNEL: $KERNEL"
     echo "  MACH: $MACH"
+    echo "  OSSTR: $OSSTR"
 
 
 
     if [[ "'$*'" =~ appimage ]] ; then
-        if [ -f $SCRIPTFOLDER/appimage.sh ]; then
+        if [ -f "$SCRIPTFOLDER/appimage.sh" ]; then
```

Comment:
We have not produced an appimage in a long time.  This in not blocking, but I think this code can either go, or be replaced with a check for snapd and then use snapd

Reply:
Adding a check for snapd sounds like a good replacement, but might not be very robust as snapd is not yet standard in a lot of distributions (I believe it's currently only shipped with Ubuntu). Because of this I would argue that deleting the code for appimage would make more sense, and implementing a snap install when Snappy is more mature/widely used.